### ISSUE TITLE
Creating a form with nested fields, an element is always automatically added

### ISSUE THREAD

**Issue by w-** [foxhound87/mobx-ajv-form#178]
if i define a form with the following fields

```
[
	'name',
	'list.cat1',
	'list.cat1[].name',
	'list.cat2',
	'list.cat2[].name',
]
```

When I create an instance of the form and i inspect the values  (or use the dev tools) i can see that on init, list.cat1 and list.cat2 automatically have an element. I would expect an empty array (or more specifically no values)

Did I miss something / is there a way i can get this behavior?

**Comment by foxhound87** [comment]
I think I can fix it. In the meantime you can remove it with the `.del()` method on form initialization.

**Comment by w-** [comment]
ok. thanks. so I basically do:

```
onInit(form){
	form.$('list.cat1').del(0);
	form.$('list.cat2').del(0);
}
```

is that right?

**Comment by foxhound87** [comment]
right 👍

**Comment by foxhound87** [comment]
I thought more on this issue and made more tests. 

It seems that keeping at least one field will prevent errors when adding deeply nested fields.

I can make this work only for the very last nested fields, but it would be not coherent with the default behavior. 

Furthermore if you don't have at least one empty field on the UI, it will confuse the user which will have to add it or you have to add it for him manually.

**Comment by foxhound87** [comment]
Presence of the first element of the array should be fixed in [version 1.19](https://github.com/foxhound87/mobx-react-form/commit/005d04c7)
